apps/challenges/permissions.py

```diff
@@ -21,3 +24,31 @@ def has_permission(self, request, view):
                 return False
         else:
             return False
+
+
+def is_challenge_creator(func):
```

Comment:
I don't think there is a need for the decorator since it is already written on line 9 of the same file. Can you please use the same decorator?